src/components/subscriptions/licenses/LicenseManagementTable/LicenseManagementTableBulkActions.jsx

```diff
@@ -287,6 +290,13 @@ LicenseManagementTableBulkActions.propTypes = {
   onRevokeSuccess: PropTypes.func.isRequired,
   onEnrollSuccess: PropTypes.func.isRequired,
   allUsersSelected: PropTypes.bool.isRequired,
+  filtersAndValues: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string,
+      filter: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
```

Comment:
Is this PropType definition accurate? Per the [react-table docs on filtering](https://react-table.tanstack.com/docs/api/useFilters), it looks like acceptable values for `filter` are either a string or a function. Not sure if `PropTypes.array` is applicable here?

Reply:
sorry those proptypes are suppose to be for filterType, not filter
<img width="337" alt="Screen Shot 2022-01-07 at 10 00 58 AM" src="https://user-images.githubusercontent.com/7385292/148562436-e046eb4c-3f3e-4b4c-822c-5d5592acb892.png">